starlite/config/compression.py

```diff
@@ -23,7 +23,7 @@ class CompressionConfig(BaseModel):
     'compression_config' key.
     """
 
-    backend: Union[CompressionBackend]
+    backend: Union[CompressionBackend, str]
     """
         [CompressionBackend][starlite.enums.CompressionBackend] or dotted path for
         compression backend to import.
```

Comment:
Is `"or dotted path for compression backend to import"` still relevant? Disregard if I'm just not looking in the right place but I can't see where that takes place - at least not local to the config type. If not, perhaps you could type this with a `Literal` instead of `str`.

Reply:
Actually, this is a good idea.  I think we can do a `PyObject` Union here to add this functionality, but will need some additional testing.  I'll make a followup PR once I test this out.  

This PR makes it so that you can use the string value from the Enum instead of the Enum itself.  (`gzip` and `brotli` in uppercase or lowercase are the only acceptable values)

Reply:
> Actually, this is a good idea.

Do you mean allowing the dotted path? What is the use case that supports?

I wasn't very clear but I was just suggesting to remove that from the docs if we don't have the functionality and typing it `backend: Union[CompressionBackend, Literal["gzip", "brotli"]]`

Reply:
I was thinking that it might be easier for a user to specify a completely custom backend this way, but perhaps not. 
Would we even need the enum of we swapped to a Literal?